test/test_functest.py

```diff
@@ -24,21 +24,26 @@ def test_functest_sanitizers():
 
 
 def check_functest(implementation):
-    helpers.make(TYPE=implementation.scheme.type,
+    helpers.make(#'functest',
+                 TYPE=implementation.scheme.type,
                  SCHEME=implementation.scheme.name,
                  IMPLEMENTATION=implementation.name,
                  working_dir=os.path.join('..', 'test'))
     helpers.run_subprocess(
-        ['./functest_{}_{}'.format(implementation.scheme.name,
-                                   implementation.name)],
+        [os.path.join('..', 'bin', 'functest_{}_{}{}'.format(
```

Comment:
Shouldn't need the ../bin added in the command, as that's been set as the working dir on line 38 below.

Reply:
That's a bug or difference in behaviour in Python on Windows and Linux, which is not documented, but without `../bin/` this does not work on Windows, *even* if you supply `cwd`.

Reply:
Sigh.